### DISCUSSION TITLE
Documentação Swagger

### DISCUSSION THREAD

**Discussion by allex-ribeiroh** [balta-io/forum#246] [Tira Dúvidas] [answered]
Balta, boa noite.

No teste que você fez na execução do Swagger para criar um account você não deveria criar primeiro um token para passar no método  e na execução sem token não deveria validar a autenticação, dando uma mensagem que o usuário não tem permissão para executar esse método?

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @allex-ribeiroh, como vai?
Qual método? Se tiver o [Authorize] ele tem que validar o token sim!
